frontend/packages/dev-console/src/utils/application-utils.ts

```diff
@@ -44,6 +47,20 @@ export const edgesFromAnnotations = (annotations): string[] => {
   return edges;
 };
 
+export const edgesFromLabels = (labels): string[] => {
```

Comment:
This assumes that by having a connects-to label that the app is bound to a service. That isn't always true. We should be getting the service binding requests and matching the selector to workloads. The connects-to label selector is a convention however any selector can be used. Then our topology wouldn't show the app binding.

Reply:
Further to this, we need a resource backing the edge itself so that we can select the edge and display the SBR in the side bar.